pyanaconda/modules/services/installation.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (C) 2019 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import os
+
+from pyanaconda.core import util
+
+from pyanaconda.modules.common.task import Task
+from pyanaconda.modules.services.constants import SetupOnBootAction
+
+from pyanaconda.anaconda_loggers import get_module_logger
+log = get_module_logger(__name__)
+
+__all__ = ["ConfigureInitialSetupTask"]
+
+
+class ConfigureInitialSetupTask(Task):
+    """Installation task for Initial Setup configuration."""
+
+    def __init__(self, sysroot, setup_on_boot):
+        """Create a new root password configuration task.
+
+        :param str sysroot: a path to the root of the target system
+        :param int setup_on_boot: setup-on-boot mode for Initial Setup
```

Comment:
But what the int here means? What will happen if I pass `-100` or `100`?
Could you please improve the documentation here?

Reply:
Indeed! Changed the type to "enum" and added a note that this is being defined by the SetupOnBoot enum. 

Doe we have something similar elsewhere ? Can definitely make it the same if we do.

Reply:
Not sure but we can change it later when we hit it.